main/src/com/google/refine/expr/functions/math/Pow.java

```diff
@@ -35,19 +35,22 @@
 
 import java.util.Properties;
 
+import com.google.refine.expr.EvalError;
+import com.google.refine.grel.ControlFunctionRegistry;
 import com.google.refine.grel.Function;
 
 public class Pow implements Function {
 
     @Override
     public Object call(Properties bindings, Object[] args) {
-        if (args.length == 2 && args[0] instanceof Number && args[1] instanceof Number) {
+        if (args.length == 2 && args[0] != null && args[0] instanceof Number
+        && args[1] != null && args[1] instanceof Number) {
```

Comment:
The null checks are redundant. `null` will never be an `instanceof Number`

Reply:
Can I use try and catch block for it.

Reply:
Hi @tfmorris before as you said about inconsistency error handling I did something which is not the proper solution but it give some relevant output
```
  if((args.length != 2)){
        return new EvalError(ControlFunctionRegistry.getFunctionName(this) + "  expects two numbers");  
        }
        else if(! (args[1] instanceof Number) || ! (args[0] instanceof Number)){
        return new EvalError(ControlFunctionRegistry.getFunctionName(this) + "   arguments is not an instance of number");
     }else{
            return Math.pow(
                ((Number) args[0]).doubleValue(),
                ((Number) args[1]).doubleValue()
                 );
        }
```
![Screenshot from 2020-08-30 21-16-34](https://user-images.githubusercontent.com/63091769/91663436-6977a000-eb06-11ea-9d2e-20280906ee7a.png)
![Screenshot from 2020-08-30 21-16-47](https://user-images.githubusercontent.com/63091769/91663440-6c729080-eb06-11ea-92f3-bbfeb0aafff3.png)
![Screenshot from 2020-08-30 21-17-07](https://user-images.githubusercontent.com/63091769/91663441-6da3bd80-eb06-11ea-8cad-7df317db955d.png)